src/django_components/expression.py

```diff
@@ -1,50 +1,55 @@
-from typing import Any, Dict, List, Mapping, Optional, Union
+from typing import Any, Dict, List, Mapping, Optional, Tuple, Union
 
-from django.template import Context
+from django.template import Context, TemplateSyntaxError
 from django.template.base import FilterExpression, Parser
 
+Expression = Union[FilterExpression]
```

Comment:
It may look strange that there's a `Union` with a single element. But this is a preparation for the dynamic expressions and spread operator syntax, which will be part of the union.